src/OmniSharp.MSBuild/Notification/ProjectLoadedEventArgs.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using OmniSharp.MSBuild.Logging;
+using MSB = Microsoft.Build;
+
+namespace OmniSharp.MSBuild.Notification
+{
+    public class ProjectLoadedEventArgs
+    {
```

Comment:
No, MSBuildProjectInfo is returned as part of the response to the `/projects` and `/project` endpoints. I added those properties because I figured that, since you'd asked for them, they'd be useful to others. You can easily grab these property values from the ProjectInstance provided.

Reply:
ProjectInstance is an MSBuild API that can be used to build particular targets for a project.

Reply:
Ah, alright sounds good. Figured i'd double check 😄